docs/release-notes/2.36.0.md

```diff
@@ -0,0 +1,124 @@
+# Percona Monitoring and Management 2.36.0
+
+| **Release date:** | Mar 21, 2023                                                                                 |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+<!---
+
+### pg_stat_monitor 2.0 support in QAN
+
+We are excited to announce that PMM V2.36 now supports [pg_stat_monitor](https://docs.percona.com/pg-stat-monitor/index.html) 2.0 (PGSM 2.0) in QAN, a powerful Query Performance Monitoring tool for PostgreSQL. With this update, you can experience the latest improvements and fixes covered by PGSM2:
+- Its improved internal architecture leads to fewer lock acquisitions, and therefore an improved performance by approximately 20% when tested using pgbench.
+- Support for PostgreSQL 15 
+- Improved consistency with pg_stat_statements - Now pg_stat_monitor view has exactly the same column names, columns and data types as the pg_stat_statements view for every major version of PostgreSQL from 11 to 15.
+- An indication of buckets as "done" vs "current" eliminates the need for the tool to evaluate the bucket status and facilitates the accurate data display
+- The generation of a unique ID of a query, independent from PostgreSQL versions, database, users or schemas, improves observability of query planning, execution and performance
+
+For more information on the enhancements included in PGSM2, please refer to the [full list] (https://github.com/percona/pg_stat_monitor/blob/main/RELEASE_NOTES.md)available.
+
+### Experimental Kubernetes Dashboards
+
+We are thrilled to introduce three new dashboards in PMM V2.36, located in the Experimental folder of the Dashboards section. These dashboards, named Kubernetes Cluster Summary, Kubernetes Pods Status, and Kubernetes Volumes, provide valuable insights into Kubernetes environments. We encourage you to test these experimental dashboards and share your feedback with us on the forum, helping us improve them for future releases.
+
+Add more subsections for more release highlights.
+
```

Comment:
```suggestion
###New Advisors presentation

Understanding what advisors you have wasn't simple in PMM. from 2.36, we are presenting checks we have in advisors grouped by Categories and Advisors.
 
###Advisors for PostgreSQL executed against all DB.

From this release, all Advisors related to PostgreSQL will be executed against all available Databases, not only the one connected to PMM. This will provide you with insights about all DBs in your PostgreSQL server. 

###Backup management is enabled by default.

Backup management in PMM is enabled by default from this release, so you only need to configure your DBs to use this feature. LNK_TO_PMM_DOC
```

Reply:
@catalinaadam ^ PTAL , the initial list